llm_demo/app.py

```diff
@@ -66,11 +68,20 @@ async def login_google(
     user_id_token = form_data.get("credential")
     if user_id_token is None:
         raise HTTPException(status_code=401, detail="No user credentials found")
+
+    user_name = get_user_name(str(user_id_token), request.app.state.client_id)
     # create new request session
     orchestrator = request.app.state.orchestration_type
     orchestrator.set_user_session_header(request.session["uuid"], str(user_id_token))
     print("Logged in to Google.")
 
+    request.session["history"][0] = {
```

Comment:
This is replacing the previous history (which is misleading). Should we either:
a. only do this if there is one message in the chat
or
b. append to the end to indicate the user is signed on?

Reply:
should we also move it into the orchestration layer -- isn't that where the history is being handled now?

Reply:
Nope, the history is being handled by the app request.